### PR TITLE
Add Getting started doc from releases

### PR DESCRIPTION
**Pull request by asifdxtreme**
https://github.com/volcano-sh/volcano/issues/347

### CODE REVIEW

File: content/en/docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+replicaset.apps/volcano-scheduler-94998fc64      1         1         1       6m10s
+
+
+
+NAME                               COMPLETIONS   DURATION   AGE
+job.batch/volcano-admission-init   1/1           28s        6m10s
+
+```
+
+Volcano scheduler utilize `queues.scheduling.incubator.k8s.io` to share resource, therefore default queue is required before usage.
```
Comment (TommyLike) [inline content/en/docs/getting-started.md]: Default queue now it's been handled in source code, therefore we don't need this step anymore.

Comment (asifdxtreme) [reply content/en/docs/getting-started.md]: I have updated the document

**Pull request merged**